code/src/layouts/root-layout/RootLayout.tsx

```diff
@@ -4,13 +4,16 @@ import Footer from "@/layouts/footer/Footer";
 import AppBox from "@/components/app-box/AppBox";
 
 import "@/layouts/root-layout/RootLayout.scss";
+import { Suspense } from "react";
 
 const RootLayout = () => {
   return (
     <AppBox className="root-layout">
       <Header />
       <AppBox className="root-layout__content">
-        <Outlet />
+        <Suspense>
```

Comment:
Please consider adding fallback, you can use AppLoader for it

Reply:
Thanks, I added fallback there